core/frontend-stubs/gradle-frontend-stub/src/main/java/org/eclipse/sw360/antenna/frontend/stub/gradle/WrappedGradleProject.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.frontend.stub.gradle;
+
+import org.eclipse.sw360.antenna.frontend.MetaDataStoringProject;
+import org.eclipse.sw360.antenna.frontend.Build;
+import org.gradle.api.Project;
+
+import java.io.File;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+public class WrappedGradleProject extends MetaDataStoringProject {
+
+    private final Project innerProject;
+    private String projectId;
+    private final File pomFile;
+
+    public WrappedGradleProject(Project innerProject, File pomFile) {
+        this(innerProject, pomFile, null);
+    }
+
+    public WrappedGradleProject(Project innerProject, File pomFile, File propertiesFile) {
+        this.innerProject = innerProject;
+        this.pomFile = pomFile;
+        this.propertiesFile = propertiesFile;
+
+        String srcDir = innerProject.getBuildFile().getParentFile()
```

Comment:
`getparentFile` may return `null`

Reply:
I changed from `getBuildFile().getParentFile()` to `getRootDir()`.